.github/workflows/changed_file_trigger_examples_check.yml

```diff
@@ -0,0 +1,109 @@
+name: Test Example
+on:
+  pull_request:
+    paths:
+      - 'examples/**'
+    # So only the changes in examples folder will trigger jobs below.
+  schedule:
+    # run at 00:00 of every Sunday(singapore time) so here is UTC time Saturday 16:00
+    - cron:  '0 16 * * 6'
+
+jobs:
+# This is for changed files detect and check.
+  detect-changed-files:
+    if: ${{ github.event_name == 'pull_request' }}
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    name: Check out all files
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 2
+      - name: Get all changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v35
+        with:
+          since_last_remote_commit: true  # Using this can trigger action each time a PR is submitted.
+      - name: setup matrix
+        id: set-matrix
+        run: |
+          changedFileName=""
+          for file in ${{ steps.changed-files.outputs.all_changed_files  }}; do
+            changedFileName="${file}:${changedFileName}"
+          done
+          echo "$changedFileName was changed"
+          res=`python .github/workflows/changed_file.py --fileNameList $changedFileName`
+          echo "All changed files are $res"
+          loc=$( IFS=',' ; echo "${res[*]}" )
+          echo "$loc"
+          echo "::set-output name=matrix::{\"loc\":$(echo "$loc")}"
+
+# If no file is changed, it will prompt an error and shows the matrix do not have value.
+  check-all-changed-files:
+    if: ${{ github.event_name == 'pull_request' }}  # Add this to avoid when schedule triggers the workflow, this should keep silent.
```

Comment:
The comment has syntax error, the correct expression is `Add this condition to avoid executing this job if the trigger event is workflow_dispatch`.

Reply:
OK, done!